### PR TITLE
CD201.md updated, MoM added 

### PR DESCRIPTION
**Pull request by accakks**

### CODE REVIEW

**Comment by maddhruv** [comment]
we will wait and add more contents and courses into the PR

**Comment by accakks** [comment]
@maddhruv @amritsrivastava MoM added .

File: 2018-19/CD201.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# CD201
+CodeDocs course 201 for session 2018-19: odd semester
+
+## Web Development
```
Comment (maddhruv) [inline 2018-19/CD201.md]: :+1:

File: 2018-19/CD201.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| Lecture 7 | Introduction to frameworks, libraries and essential tools |
+-----
+
+## Android Development
+| Lecture | Topics | 
+| ----- | ----- |
+| Lecture 0 |Getting started with Android and Android architecture | 
+| Lecture 1 | Basics of JAVA |
+| Lecture 2 | Setting up Android SDK and First Project | 
+| Lecture 3 | Building a basic Android app |
```
Comment (maddhruv) [inline 2018-19/CD201.md]: Lecture 3 += 4;

File: 2018-19/CD201.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+## Android Development
+| Lecture | Topics | 
+| ----- | ----- |
+| Lecture 0 |Getting started with Android and Android architecture | 
+| Lecture 1 | Basics of JAVA |
+| Lecture 2 | Setting up Android SDK and First Project | 
+| Lecture 3 | Building a basic Android app | 
+| Lecture 4 | Activities | 
+| Lecture 5 | Services and UI of Android |
+| Lecture 6 | Data Storage in Android |
```
Comment (maddhruv) [inline 2018-19/CD201.md]: SQLite & SharedPreferences

File: 2018-19/CD201.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| ----- | ----- |
+| Lecture 0 |Getting started with Android and Android architecture | 
+| Lecture 1 | Basics of JAVA |
+| Lecture 2 | Setting up Android SDK and First Project | 
+| Lecture 3 | Building a basic Android app | 
+| Lecture 4 | Activities | 
+| Lecture 5 | Services and UI of Android |
+| Lecture 6 | Data Storage in Android | 
+| Lecture 7 | Android APIs and JSON parsing | 
+
```
Comment (maddhruv) [inline 2018-19/CD201.md]: Lecture 8: Firebase & RecyclerView ;
Lecture 9: _something_

File: 2018-19/CD201.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+| Lecture 1 | HTML - Structuring the web | 
+| Lecture 2 | CSS - Styling the web | 
+| Lecture 3 | JavaScript | 
+| Lecture 4 | Dynamic client-side scripting | 
+| Lecture 5 | Server side website programming | 
+| Lecture 6 | Browser APIs and Third party APIs | 
+| Lecture 7 | Introduction to frameworks, libraries and essential tools |
+-----
+
+## Android Development
```
Comment (maddhruv) [inline 2018-19/CD201.md]: :+1:

File: 2018-19/CD201.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+| Lecture 5 | Services and UI of Android |
+| Lecture 6 | Data Storage in Android | 
+| Lecture 7 | Android APIs and JSON parsing | 
+
+-----
+
+## Extra 
+(Independent sessions or talks)
+
+### Git & GitHub `1`
```
Comment (maddhruv) [inline 2018-19/CD201.md]: :+1:

File: 2018-19/CD201.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+-Intro to Git and GitHub 
+-Diving deep into Git 
+
+### Linux sessions
+- Shell programming (basics)
+- How shell scripting makes your life easier (Hands on automating with shell scripting)
```
Comment (maddhruv) [inline 2018-19/CD201.md]: > Hands on automating with shell scripting 

:+1:

File: 2018-19/CD201.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+-----
+
+## Extra 
+(Independent sessions or talks)
+
+### Git & GitHub `1`
+-Intro to Git and GitHub 
+-Diving deep into Git 
+
+### Linux sessions
```
Comment (maddhruv) [inline 2018-19/CD201.md]: :+1:

File: 2018-19/CD201.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+- Shell programming (basics)
+- How shell scripting makes your life easier (Hands on automating with shell scripting)
+.... add more
+
+### Python `Abandoned`
```
Comment (maddhruv) [inline 2018-19/CD201.md]: :+1:

**Comment by accakks** [comment]
https://github.com/CodeDocsJECRC/CD201/pull/3

**Pull request merged**